pkg/siteconfig/clusterinstance_test.go

```diff
@@ -21,6 +28,62 @@ var testSchemes = []clients.SchemeAttacher{
 	siteconfigv1alpha1.AddToScheme,
 }
 
+func TestNewClusterInstanceBuilder(t *testing.T) {
+	generateCIBuilder := NewCIBuilder
+
+	testCases := []struct {
+		name              string
+		namespace         string
+		client            bool
+		expectedErrorText string
+	}{
+		{
+			name:              testClusterInstance,
+			namespace:         testClusterInstance,
+			client:            true,
+			expectedErrorText: "",
+		},
+		{
+			name:              "",
+			namespace:         testClusterInstance,
+			client:            true,
+			expectedErrorText: "clusterinstance 'name' cannot be empty",
+		},
+		{
+			name:              testClusterInstance,
+			namespace:         "",
+			client:            true,
+			expectedErrorText: "clusterinstance 'nsname' cannot be empty",
+		},
+		{
+			name:              testClusterInstance,
+			namespace:         testClusterInstance,
+			client:            false,
+			expectedErrorText: "",
+		},
+	}
+
+	for _, testCase := range testCases {
+		var testSettings *clients.Settings
+
+		if testCase.client {
+			testSettings = clients.GetTestClients(clients.TestClientParams{})
+		}
+
+		testClusterInstanceStructure := generateCIBuilder(
+			testSettings,
+			testClusterInstance,
+			testClusterInstance)
```

Comment:
these should be `testCase.name` and `testCase.namespace`, respectively, which should fix the test failure